dashboard/app/asset_storage.go

```diff
@@ -512,6 +512,13 @@ func createAssetList(build *Build, crash *Crash, forReport bool) []dashapi.Asset
 			DownloadURL: reportAsset.DownloadURL,
 			Type:        reportAsset.Type,
 		})
+		if reportAsset.FsckLog != 0 {
```

Comment:
I'd focus on extending the `dashapi.Asset` type instead. If we store fsck logs as separate assets, we'd also have to correlate them with the mounted images, which will only complicate UI/email template rendering.